### DISCUSSION TITLE
Saving to Commander Core XT Onboard Memory

### DISCUSSION THREAD

**Discussion by fraibert** [EvanMulawski/FanControl.CorsairLink#128] [Q&A] [answered]
Hi,

I'm interested in using Fan Control instead of iCUE to control the fans attached to a Corsair Commander Core XT.  I have an AIO that does not have a temperature probe (the EK Nucleus) so I am using one of the Commander's temperature probes to track the AIO's liquid temperature by placing the probe just behind the radiator.  Using iCUE, I have a fan curve set based upon the liquid temperature as read by the probe.

My understanding is that the Corsair Commanders, if set in iCUE to use a fan curve based on a temperature probe, will save that curve into the hardware.  Is this behavior possible to reproduce on using Fan Control with the CorsairLink plug in?  (I prefer to avoid having fan control dependent on any software.)

Thank you!

**Comment by EvanMulawski** [comment, marked as answer]
Hi @fraibert, this plugin does not support saving any settings/curves to the device. You would need to use iCUE to save that information to the device to allow it to run in hardware mode.

**Reply by fraibert** [reply]
Thank you for the prompt reply.  I'm sorry to be a bother about this, but do you think it's a possible feature in the future?  iCUE is not my favorite piece of software...

**Reply by EvanMulawski** [reply]
Not within this plugin - it would be a separate program/utility.

**Reply by fraibert** [reply]
I appreciate your time in answering the question.  Thank you also for your work on the plug in!
